[weston] Unable to install

```
checking for wayland-protocols >= 1.13... no
configure: error: Package requirements (wayland-protocols >= 1.13) were not met:

Requested 'wayland-protocols >= 1.13' but version of Wayland Protocols is 1.9

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables WAYLAND_PROTOCOLS_CFLAGS
and WAYLAND_PROTOCOLS_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
weston failed to build: `./autogen.sh` exited with 1
```

The error above was fixed with `crew upgrade wayland_protocols`, however, a new error occurred afterwards:
```
/bin/grep: /usr/local/lib/libdrm.la: No such file or directory
/bin/sed: can't read /usr/local/lib/libdrm.la: No such file or directory
libtool:   error: '/usr/local/lib/libdrm.la' is not a valid libtool archive
make[1]: *** [Makefile:4029: libshared-cairo.la] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/3.0.91.tar.gz.dir/weston-3.0.91'
make: *** [Makefile:3560: all] Error 2
weston failed to build: `make V=0 -j2` exited with 2
```
This appears to be a case where `weston` is looking for `libdrm.la` in the wrong directory.  The correct location is `/usr/local/lib64/libdrm.la`.

--- a/packages/weston.rb
+++ b/packages/weston.rb
@@ -7,6 +7,19 @@ class Weston < Package
   source_url 'https://github.com/wayland-project/weston/archive/3.0.91.tar.gz'
   source_sha256 'a27eecd9dd9a18131292d33eba37f35369ede5f3b0688154244e3238b27f854a'
 
+  binary_url ({
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/weston-3.0.91-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/weston-3.0.91-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/weston-3.0.91-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/weston-3.0.91-chromeos-i686.tar.xz',
+  })
+  binary_sha256 ({
+     x86_64: '6ce064ae0d5b4282858d436d24e902f2f690d2bb7f1acca576013c201c4b9c42',
+    aarch64: '07fd4523ff123b53271ac100d5b2bee8edd40a1e11026cbfc19ab2671179f7f4',
+     armv7l: '07fd4523ff123b53271ac100d5b2bee8edd40a1e11026cbfc19ab2671179f7f4',
+       i686: '1bf7ac234043bfad8877e0eea7f58ca20487d1464a865e2ab12e877404ebe8e1',
+  })
+
   depends_on 'harfbuzz'
   depends_on 'libxcursor'
   depends_on 'libinput'

